frontend/components/MockTips.json

```diff
@@ -0,0 +1,80 @@
+[
+  {
+    "tipID": "1",
+    "tipSubject": "Give your plants appropriate lighting",
+    "tipMessage": "Different kinds of houseplants have different lighting requirements, so be mindful about where you put your plants.",
+    "sourceURL": "https://www.bioadvanced.com/articles/tips-healthy-houseplants"
```

Comment:
Do we want to add plantType?

Reply:
I'm not sure. Some tips can be specific to a plantType, but a lot of tips are general for a variety of plant types. Perhaps have a plantType field that specifies an array of plant types, and use null to indicate "this tip applies to all plant types"?